python/adelphi/adelphi/anonymize.py

```diff
@@ -0,0 +1,96 @@
+# Functions and constants related to the anonymization process
+
+from adelphi.store import get_standard_columns_from_table_metadata
+
+# default prefixes for the anonymized names
+KEYSPACE_PREFIX = "ks"
+TABLE_PREFIX = "tbl"
+PARTITION_KEY_PREFIX = "pk"
+CLUSTERING_KEY_PREFIX = "ck"
+COLUMN_PREFIX = "col"
+TYPE_PREFIX = "udt"
+FIELD_PREFIX = "fld"
+INDEX_PREFIX = "idx"
+
+# maps the original schema names to the replacement names
+name_map = {
+    KEYSPACE_PREFIX: {},
+    TABLE_PREFIX: {},
+    PARTITION_KEY_PREFIX: {},
+    CLUSTERING_KEY_PREFIX: {},
+    COLUMN_PREFIX: {},
+    TYPE_PREFIX: {},
+    FIELD_PREFIX: {},
+    INDEX_PREFIX: {}
+}
+
+
+def get_name(original_name, prefix):
+    """
+    Looks up the anonymized name for the provided original name in the cache.
+    If not present, one is created, inserted into the cache and returned.
+    """
+    count = len(name_map[prefix])
+    anonymized_named_prefixed = "%s_%s" % (prefix, count)
+    return name_map[prefix].setdefault(original_name, anonymized_named_prefixed)
+
+
+def anonymize_keyspace(keyspace):
+    keyspace.name = get_name(keyspace.name, KEYSPACE_PREFIX)
+
+    for table in keyspace.tables.values():
+        anonymize_table(table)
+
+    for table in keyspace.user_types.values():
```

Comment:
nit: this should be `for udt` (noticed this in my branch too)

Reply:
Ah, good catch!  I'll fix this as well.